test_runner/random_test_runner/include/random_test_runner/lanelet_utils.hpp

```diff
@@ -58,11 +59,17 @@ class LaneletUtils
     const traffic_simulator_msgs::msg::LaneletPose & pose, double min_distance,
     double max_distance);
 
+  std::vector<geometry_msgs::msg::Pose> getPositionsOnNextLanelets(
+    const traffic_simulator_msgs::msg::LaneletPose & start_lanelet,
+    const int64_t & max_number_of_points, double offset = 0.0);
   std::vector<int64_t> getLaneletIds();
+  lanelet::Ids filterLaneletIds(const lanelet::Ids & lanelet_ids, const char subtype[]);
   geometry_msgs::msg::PoseStamped toMapPose(traffic_simulator_msgs::msg::LaneletPose lanelet_pose);
   std::vector<int64_t> getRoute(int64_t from_lanelet_id, int64_t to_lanelet_id);
   double getLaneletLength(int64_t lanelet_id);
   bool isInLanelet(int64_t lanelet_id, double s);
+  std::optional<traffic_simulator_msgs::msg::LaneletPose> toLaneletPose(
+    geometry_msgs::msg::Pose global_pose, bool include_crosswalk = false);
```

Comment:
A function is implemented that simply calls a member of the HdMapUtils class.
The LaneletUtils::toLaneletPose function is called only once inside LaneletUtils, so there is no need to make it a function.

Even if you must make it a function, avoid unnecessary copying.

Reply:
The function was needed in the [test_randomizer.cpp](https://github.com/tier4/scenario_simulator_v2/pull/1151/files#diff-0349381497ffdc3771f21d77972e127ae4056d912db6a2a9e6ce786f85216d29R345). However I will look into the way how to simplify the usage of LaneletUtils and see if it is possible to merge with  traffic_simulator::HdMapUtils